services/thermal_mgr/thermal_mgr.c

```diff
@@ -43,18 +44,54 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
   /* Send an event to the thermal manager queue */
-
-  return ERR_CODE_SUCCESS;
+    if (event == NULL) {
+        return ERR_CODE_INVALID_QUEUE_MSG;}
+    if (xQueueSend(thermalMgrQueueHandle, (void *)event, (TickType_t) 0) == pdTRUE){
+        return ERR_CODE_SUCCESS;
+    }
+  return ERR_CODE_UNKNOWN; // prev: ERR_CODE_QUEUE_FULL;
```

Comment:
If you check that the queue has been created this error code can just be `ERR_CODE_QUEUE_FULL` since that's the only other way this would fail